playbook.yaml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Configure App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: TRELLO_API_KEY
+      prompt: Trello API Key?
+
+    - name: TRELLO_API_TOKEN
+      prompt: Trello API Token?
+
+    - name: TRELLO_BOARD_ID
+      prompt: Trello Board ID?
+      default: 65e4d3b571aacb487f9dc1bf
+
+    - name: TRELLO_TO_DO_LIST_ID
+      prompt: Trello "To-Do" List ID?
+      default: 65e4d3b571aacb487f9dc1c6
+
+    - name: TRELLO_DOING_LIST_ID
+      prompt: Trello "Doing" List ID?
+      default: 65e4d3b571aacb487f9dc1c7
+
+    - name: TRELLO_DONE_LIST_ID
+      prompt: Trello "Done" List ID?
+      default: 65e4d3b571aacb487f9dc1c8
+
+  tasks:
+    - name: Install Dependencies
+      become: true
+      yum:
+        name:
+          - git
+          - python3
+        state: present
+
+    - name: Install Poetry
+      shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: ~/.local/bin/poetry
+
+    - name: Create App Directory
+      become: true
+      file:
```

Comment:
Aha, one of the rare people that skip the references to `ansible.builtin` - I like it!